resources/asciidoctor/lib/docbook_compat/convert_table.rb

```diff
@@ -53,6 +53,7 @@ def convert_table_tag(node, border)
         %( border="#{border}" cellpadding="4px"),
         node.title ? %( summary="#{strip_tags node.title}") : nil,
         (width = node.attr 'width') ? %( width="#{width}") : nil,
+        node.role ? %( class="#{node.role}") : nil,
```

Comment:
Is there a chance you'd like to write a test for this one? At this point we're not great at maintaining all this, but a test would help to make sure that the next person who touches this doesn't break it. The tests are all hiding in `resources/asciidoctor/spec/docbook_compat_spec.rb` and `make check` should just run them all.